### DISCUSSION TITLE
感觉可以塞一个纯本地的教程

### DISCUSSION THREAD

**Discussion by dongrentianyu** [tiddly-gittly/TidGi-Desktop#587] [Ideas]
![image](https://github.com/user-attachments/assets/d46f1590-ebe0-419a-beaa-2fbd74b70fd4)

类似图片中的这种，把tiddlywiki-xp搞出来的一些基本知识放里面去。当然我感觉只搞中文就行。如果要英文连着一起搞的话，就会有点麻烦。

**Comment by linonetwo** [comment]
现在已经有在线教程了，而且有好多个，所以没必要搞本地的了。现在缺的是往太记官网上多写一点插件教程，我准备用截图的形式来做，方便之后黏贴到 pkmer 之类的网站。
